### PR TITLE
Fix & extend CodeGraphic. Adds symbols, strings, etc

### PR DESCRIPTION
**Pull request by nezza**
It also adds a cursor and makes the whole thing clickable to jump through the file. Should also help with #167.

screen shot 2017-12-07 at 5 38 38 pm

### CODE REVIEW

**Comment by nezza** [comment]
Adjusted the colors a bit to have more contrast.
screen shot 2017-12-07 at 8 38 38 pm

**Comment by radare** [comment]
❤️

File: src/widgets/CodeGraphic.cpp [hunk 1]
```diff
@@ -51,8 +53,36 @@ void GraphicsBar::paintEvent(QPaintEvent *event)
...
+        QJsonObject jObject = doc.object();
+        QVariantMap mainMap = jObject.toVariantMap();
+        blockMaps.append(mainMap);
+        totalSectionsSize += section.size;
+    }
+}
+
 void GraphicsBar::fillData()
 {
+    int from = blockMaps.first()["from"].toInt();
```
Comment (xarkes) [inline src/widgets/CodeGraphic.cpp]: .toULongLong() seems better no?

**Comment by xarkes** [comment]
Looks very nice, thanks!

File: src/widgets/CodeGraphic.cpp (hunk 1 above)
Comment (nezza) [reply src/widgets/CodeGraphic.cpp]: Yep, after what I've seen in #188 this needs to be toVariant().toULongLong(). I'll fix it in a bit.

Comment (xarkes) [reply src/widgets/CodeGraphic.cpp]: Ah sorry I merged already.

Comment (xarkes) [reply src/widgets/CodeGraphic.cpp]: You can use the pen directly on GitHub

**Pull request merged**